### PR TITLE
DEV: Add linting and CI tests

### PR DESCRIPTION
**Pull request by jmperez127**

### CODE REVIEW

File: assets/javascripts/discourse/controllers/webinar-picker.js.es6 [hunk 1]
```diff
@@ -131,8 +131,8 @@ export default Controller.extend(ModalFunctionality, {
     return false;
   },
 
-  @discourseComputed("pastWebinarTitle", "pastStartDate")
-  pastWebinarDisabled(title, startDate) {
+  @discourseComputed
+  pastWebinarDisabled() {
```
Comment (CvX) [inline assets/javascripts/discourse/controllers/webinar-picker.js.es6:135]: This depends on those two properties. The fix would be to use those function args instead. 🙂

File: app/controllers/webinars_controller.rb [hunk 2]
```diff
@@ -89,7 +88,7 @@ def update_nonzoom_host
         WebinarUser.where(webinar: webinar, type: :host).destroy_all
         WebinarUser.create!(user: user, webinar: webinar, type: :host)
         render json: success_json
-        return
+        nil
```
Comment (CvX) [inline app/controllers/webinars_controller.rb:91]: Those two `nil` values are not necessary

File: assets/javascripts/discourse/components/webinar-register.js.es6 [hunk 3]
```diff
@@ -82,16 +82,16 @@ export default Component.extend({
     return false;
   },
 
-  @discourseComputed("webinar.starts_at", "webinar.attendees")
-  canUnregister(starts_at, attendees) {
+  @discourseComputed("webinar.starts_at")
+  canUnregister(starts_at) {
```
Comment (CvX) [inline assets/javascripts/discourse/components/webinar-register.js.es6:86]: This should declare dependency on `isAttendee` and `registered`. The same applies to `userCanRegister()` below.

**Review by CvX — CHANGES_REQUESTED** [review]

**Comment by pmusaraj** [comment]
Specs are fixed here now @jmperez127, I'll let you address CvX's comments.

**Review by CvX — APPROVED** [review]
I fixed issues mentioned in my comments, and updated files from the plugin skeleton. Merging!

**Pull request merged**